app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -100,8 +101,8 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
       if (!tx.ctbId) captureMessage(`Tx without hash: ${JSON.stringify(tx)}`)
       return prepareTxHistoryEntry(tx, addresses)
     })
-
-    return txHistoryEntries.sort((a, b) => b.ctbTimeIssued - a.ctbTimeIssued)
+    const validTransactions = filterValidTransactions(txHistoryEntries)
```

Comment:
I'd add a TODO comment along the lines of `// TODO temporary workaround until collateral inputs in adalite-backend are exposed to properly show effect of invalid transactions in UI/csv export`